mypy/test/teststubgen.py

```diff
@@ -266,6 +266,7 @@ def run_case(self, testcase: DataDrivenTestCase) -> None:
             with open(file, 'w') as f:
                 f.write(content)
 
+        sys.path.append('.')
```

Comment:
Why is this change needed? Is it here so that tests mimic the logic of `mypy.stubgen.main()`? Please add a comment.

Reply:
current folder is not in sys path, when I test the function generate_stubs with `--module` option which should call `importlib.import_module` can't find the module. Thus I add it to sys.path.